source/sass/31-molecules/map/_leaflet.scss

```
@@ -105,4 +105,39 @@
     border-style: solid;
     border-width: 1px;
   }
+
+  .gent-leaflet-location-icon {
+    @include theme('background-color', 'color-none', 'leaflet-map-location-icon-background-color');
+
+    border-radius: 30px;
+    box-shadow: 0 3px 14px rgba(0, 0, 0, 0.4);
```

Remove leading zeros.
```suggestion
    box-shadow: 0 3px 14px rgba(0, 0, 0, .4);
```